full-moon/src/tokenizer/structs.rs

```diff
@@ -121,21 +120,28 @@ symbol! {
         Until => "until",
         While => "while",
 
-        [lua52 | luajit] Goto => "goto",
+        [lua52 | luajit | cfxlua] Goto => "goto",
```

Comment:
If you make the above change to the bitfield, you won't need to change this
```suggestion
        [lua52 | luajit] Goto => "goto",
```